x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/__snapshots__/overrides.test.js.snap

```diff
@@ -2,6 +2,28 @@
 
 exports[`Overrides render overrides 1`] = `
 <EuiForm>
+  <EuiFormRow
+    describedByIds={Array []}
+    error="Value must be greater than 3 and less than or equal to 1000000"
```

Comment:
Isn't it possible to localize this error message?

Reply:
in the snapshot file? i don't know?

Reply:
Looking in other snapshots, seems only the `label` is localized, and not `error` text,

Reply:
Doesn't look like it based on already translated snapshot files:
https://github.com/elastic/kibana/blob/735cc82edd267dab9dcdaff739875ec54cd3c977/x-pack/plugins/ml/public/settings/calendars/edit/calendar_form/__snapshots__/calendar_form.test.js.snap#L32